guide/qip/qip-simulator.rst

```diff
@@ -0,0 +1,318 @@
+.. QuTiP
+   Copyright (C) 2011-2012, Paul D. Nation & Robert J. Johansson
+
+.. _qip_simulator:
+
+****************
+Exact Simulation
+****************
+
+Let's start off by defining a simple circuit which we use to demonstrate a few
+examples of circuit evolution. First, we read in the circuit from a **.qasm** file using
+the :func:`qutip.qip.qasm.read_qasm()` function.
```

Comment:
It would be nice to start by explaining what is the exact simulator.  Like:

> :class:`~qutip.qip.QubitCircuit` uses this simulator internally to compute the result of a quantum circuit. It generates corresponding unitary for each gate and represents quantum states either as a vector or as a density matrix. Here, the circuit is assumed to be noiseless.

Besides, it is best to avoid importation from `qasm` at the beginning. Just define the circuit in the QuTiP way.  `qasm` importation and exportation should be introduced somewhere else.